deploy/README.md

```diff
@@ -17,7 +17,7 @@ All the configuration files are within the `deploy/config` directory. It is nece
 * The `postgres.env`, to define a password to the database;
 * The `pgadmin.env`, to define an the database admin credentials;
 * Assign the previously defined database password in the `spring.datasource.password` property on `application.properties` file;
-* Configure the `arrebol.json` file to tune Arrebol internals.
+* Configure the `arrebol.json` file to tune Arrebol internals. In it you will write the worker nodes ip.
```

Comment:
You will write the worker nodes ip there

Reply:
done.